### DISCUSSION TITLE
the victory music keeps playing a loop.

### DISCUSSION THREAD

**Discussion by GreatFunk** [ikemen-engine/Ikemen-GO#2080] [Feature Request]
### Describe the bug

idk why, but it's strange for me.

### To Reproduce

1. download stage from mfg.
2. start a match (like watch mode and vs mode)
3. press f1 until the victory music
4. and this happens.

### Expected behavior

i was expecting a normal victory music but then it is a loop.

### Screenshots / Video

https://github.com/user-attachments/assets/630238ad-40db-4539-ba2b-6b9988defc46

### Engine Version (or source code date)

nightly 12/10/24

### Operating system

Windows

**Comment by potsmugen** [comment]
Do you have `victory.bgm.loop = 0` in the screenpack? I just confirmed that it stops correctly.

Oh nevermind. It's the stage's `bgmusic.victory`. It looks like we don't have a loop flag parameter for those?

**Comment by K4thos** [comment]
works as expected, not an issue
